tests/std/tests/P0466R5_layout_compatibility_and_pointer_interconvertibility_traits/test.cpp

```diff
@@ -79,17 +97,22 @@ constexpr bool test() {
         class C : public A {
             int : 0;
         };
+        class D : public C {};
 // Disable warning C4408: anonymous union did not declare any data members
 #pragma warning(push)
 #pragma warning(disable : 4408)
-        class D : public A {
+        class E : public A {
             union {};
         };
 #pragma warning(pop)
-        class A1 : public A {};
-        class A2 : public A {};
-        class A3 : public A2 {};
-        class A4 : public A1, public A3 {};
+        class F : private A {}; // Non-public inheritance
+        class NS : public B, public C {}; // Non-standard

```

Comment:
All of these types are non-standard, the standard doesn't define them. I think you mean non-standard-layout. (Also on 183 and 228.)

Reply:
Yes, non-standard layout. I'll update the comments.